### ISSUE TITLE
Share to Dm 

### ISSUE THREAD

**Issue by Gottarocket** [austinhuang0131/barinsta#1497]
I appreciate new update adding option to share post as dm but we also need option to share any link, media on share option choosing barinsta chats option for now if we share using external apps it shows Error not found 404

**Comment by austinhuang0131** [comment]
yes, because that share option is an "Open in Barinsta" option, which we might eventually remove
